client/app/components/Input/__tests__/InputDefault.spec.jsx

```diff
@@ -18,9 +19,13 @@ describe('InputDefault', () => {
     jest.spyOn(window, 'alert');
   });
 
+  afterEach(() => {
+    jest.clearAllMocks();
```

Comment:
Is this necessary?

Reply:
Similarly I think I added this reset as a way to future-proof any extended tests relying on `window.alert`, which might accidentally pass when they shouldn't because of a separate call.

Should that situation be handled when the time comes? I'm worried someone might roll with a false positive. But that can be caught in code review, so I can still definitely remove this.